testing-internal/src/main/resources/junit-platform.properties

```diff
@@ -1 +1,2 @@
+junit.jupiter.execution.timeout.mode = disabled_on_debug
```

Comment:
Nice 👍

Reply:
How about setting the default timeout for all testables and life cycle methods?

```
junit.jupiter.execution.timeout.default = 60s
```

Then we could remove all `@Timeout` annotations.

Reply:
Added, thanks for the suggestion!